pkg/models/devops/json.go

```diff
@@ -729,3 +730,329 @@ type Crumb struct {
 	Crumb             string `json:"crumb"`
 	CrumbRequestField string `json:"crumbRequestField"`
 }
+
+// CheckScriptCompile
```

Comment:
for serializable struct, add `omitempty` to skip field generation to reduce response size